src/models/changeinfos.js

```diff
@@ -0,0 +1,27 @@
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class changeInfos extends Model {
```

Comment:
What about change this to Pascal case

Reply:
this file was created automatically, if we need to change this, so after each generated modelwill need to change it, are you sure that need to change it?

Reply:
We should support one style in each part in application.
In this case, all models needs to be defined in one style, in Pascal case.
I don't think that it's a big problem to rerun `model:generate` again with same params, and regenerate these few files

Reply:
So just regenerate to ChangeInfos and anothers file regenerate same with PascalCase?

Reply:
Yes. If there are any other models with camelCase or lowercase, please regenerate them as well

Reply:
Done